### ISSUE TITLE
高斯分类识别问题

### ISSUE THREAD

**Issue by gaoxiaoyao** [cornellius-gp/gpytorch#1182]
你好，我是学习高斯过程的一名新手，用高斯分类的方法做mnist数字识别研究。现在训练完模型想要输出模型的识别精度，但在输出时遇到了困难，提示张量问题，可以帮忙看一下怎样修改输出精度么，谢谢你的建议。为方便起见，我将压缩程序放在下面
num_epochs=3
for i in range(num_epochs):
    scheduler.step()
    for minibatch_i, (x_batch, y_batch) in enumerate(train_loader):
       
        optimizer.zero_grad()
        output = model(x_batch)
        log_lik, kl_div, log_prior = mll(output, y_batch)
        loss = -(log_lik - kl_div + log_prior)
       
        print('Epoch %d [%d/%d] - Loss: %.3f [%.3f, %.3f, %.3f]' % (i + 1, minibatch_i, len(train_loader), loss.item(), log_lik.item(), kl_div.item(), log_prior.item()))
        loss.backward()
        optimizer.step()
        
[multiclass_classifier_for_issue.zip](https://github.com/cornellius-gp/gpytorch/files/4798814/multiclass_classifier_for_issue.zip)

**Comment by gpleiss** [comment]
I'm sorry @gaoxiaoyao - would you be able to re-post this issue in English?
